src/core/manager.c

```diff
@@ -1729,13 +1729,20 @@ static void manager_ready(Manager *m) {
         /* Let's finally catch up with any changes that took place while we were reloading/reexecing */
         manager_catchup(m);
 
+        /* Create a file which will indicate when the manager started loading units the last time. */
+        (void) touch_file("/run/systemd/systemd-units-load", false,
+                m->timestamps[MANAGER_TIMESTAMP_UNITS_LOAD].realtime ?: now(CLOCK_REALTIME),
```

Comment:
We usually not allow implicit cast to boolean:
```
m->timestamps[MANAGER_TIMESTAMP_UNITS_LOAD].realtime > 0 ?
m->timestamps[MANAGER_TIMESTAMP_UNITS_LOAD].realtime : now(CLOCK_REALTIME),
```

Reply:
But personally, I prefer the current form. Maybe, we should relax our coding style and allow implicit cast to boolean in ternary operation?

Reply:
> We usually not allow implicit cast to boolean

I didn't know about that, but If it's expected, I'll be happy to update the code accordingly.